bindings/rust/wasmedge-sys/src/utils.rs

```diff
@@ -1,10 +1,8 @@
 use crate::error::WasmEdgeResult;
-use std::{ffi::CString, path::Path};
+use std::{ffi::CString, os::unix::ffi::OsStrExt, path::Path};
 
 #[cfg(unix)]
 pub fn path_to_cstring(path: &Path) -> WasmEdgeResult<CString> {
-    use std::os::unix::ffi::OsStrExt;
```

Comment:
I am not sure if we do need to move the unix only crate to the top?

Reply:
Accepted. Keeping it inside the function is more reasonable. Thanks a lot!